stable/kommander/values.yaml

```diff
@@ -1,6 +1,38 @@
-kommander:
-  image:
-    repository: mesosphere/kommander
-    tag: 1.24.2
-    pullPolicy: IfNotPresent
-  replicas: 1
+image:
+  repository: mesosphere/kommander
+  tag: 1.24.2
+  pullPolicy: IfNotPresent
+replicas: 1
+
+extraInitContainers:
+
+resources:
+  requests:
+    memory: "128Mi"
+    cpu: "1000m"
+  limits:
+    memory: "512Mi"
+    cpu: "4000m"
```

Comment:
this is really high for kommnader, let's put 2000 although I think we should check what it uses on regular basis..

Reply:
Changed.